features/api/metrics/index.feature

```diff
@@ -34,6 +34,10 @@ Feature: Product > Integration > Metrics
       Then I should be able to add a mapping rule to "Diavola"
       But I should see "Carbonara" already mapped
 
+    @search
+    Scenario: Sorting methods
```

Comment:
Hm... What do you mean by "sorting" here?
Is it filtering? "sorting" normally is "ordering"

Reply:
Nice catch! Changed! :)